src/components/usa-button/content/usa-button~outline-inverse.json

```diff
@@ -1,5 +1,6 @@
 {
   "modifier": "usa-button--outline usa-button--inverse",
   "title": "Outline inverse button",
-  "show_states": true
+  "show_states": true,
```

Comment:
This is the only story that has a "show_states" control. It doesn't appear to do anything. Can we remove?